crates/python/qcs_sdk/qvm/__init__.pyi

```diff
@@ -13,6 +13,10 @@ class QVMClient:
     def new_http(endpoint: str) -> QVMClient:
         """Construct a new client which uses HTTP to communicate with QVM"""
         ...
+    @staticmethod
+    def new_libquil() -> QVMClient:
+        """Construct a new client which uses libquil to communicate with QVM"""
```

Comment:
To "communicate with" QVM, or as the QVM implementation itself?

Reply:
`libquil` is not the implementation, but the bridge _to_ the implementation (QVM)

Reply:
Doesn't the `libquil` library contain the QVM code, though? Reading this, I would understand the comment to mean that I still need a QVM service running, but it uses a different protocol to communicate with the service, whereas my understanding is that `libquil` _contains_ (i.e. links to) the QVM code itself.

Maybe this would be better?

 ```suggestion
        """Construct a new client which uses libquil to provide the QVM"""
```